google/gmail/listInbox.py

```diff
@@ -8,7 +8,12 @@ async def main():
     max_results = os.getenv('MAX_RESULTS', '100')
     if max_results is not None:
         max_results = int(max_results)
-    query = 'label:inbox'
+    category = os.getenv('CATEGORY', 'primary')
+    label = os.getenv('LABEL', 'inbox')
```

Comment:
Can we validate that the values of category and label are among the allowed options?
for example : `if "label" is not in ['inbox', 'starred', 'important', 'sent', 'drafts', 'trash', 'spam', 'all']`

Reply:
We have a set of options here, but gmail allows you to apply custom labels too (I use the feature to pretty extensively myself), so it may make sense to make this less restrictive as well.

Reply:
I see.  if custom labels are allowed then we don't need to validate it here.